### PR TITLE
Added a tab for ongoing studies

### PR DESCRIPTION
**Pull request by cabreraalex**
@venkatesh-sivaraman and I chatted about adding a tab for ongoing studies we can point people to. Here's a first draft of what it could look like.
Screen Shot 2021-02-20 at 1 15 49 PM

### CODE REVIEW

**Comment by domoritz** [comment]
Three things.
1. Did you test this on a small screen?
2. How do we ensure that we don't have old studies on the website? Do you want to feel responsible for keeping this section up to date? Should we add an expiration date?
3. Why should this be a top level item in the menu? Courses/News aren't there either so we could instead list studies on the homepage similar to classes and news. If you think we only add this section temporarily, should we maybe just write a "news" post for the studies to ask for participants? We could also post the articles on our Twitter. If you think that makes sense, we could instead put news into the main menu (so far we didn't have enough content to really justify it).

**Review by domoritz — CHANGES_REQUESTED** [review]
I think it would be better to add studies as news to reduce the complexity of the navigation. We could then add news to the top level menu.

**Comment by cabreraalex** [comment]
I like that suggestion, paging @venkatesh-sivaraman for thoughts, he thought up the original plan!

**Comment by venkatesh-sivaraman** [comment]
I haven’t looked at the code yet so can’t comment too thoroughly, but I really like the idea about adding a study date range and automatically graying it out when it’s completed.

I do think it’s nice to have a separate conceptual page for recruiting, since it’s meant for a slightly different audience than general news. Also part of the goal (I don’t think it’s implemented yet) was to create a single form that a data scientist could submit in order to be notified about new studies they would be eligible for, and this link would have to go at the top of the page. Plus I think the typical expectation of a user on a lab’s News page is not that they’re going to find current/actionable information.

As for why it should be included in the top-level menu, I’m ambivalent but my thought is that this is essentially advertising, and more paths to the same page is probably better...

**Comment by domoritz** [comment]
Good points. I would still vote for making the study calls part of the news as they would for example appear in people's RSS readers. Study calls can have specific tags/metadata (https://github.com/cmudig/cmudig.github.io/blame/master/_posts/2019-10-15-join-us.md#L4) that we could render differently with https://github.com/cmudig/cmudig.github.io/blob/master/_layouts/post.html.

**Comment by cabreraalex** [comment]
also paging @adamperer

**Comment by domoritz** [comment]
Sounds great. 

> Only question I have now is do we still auto-remove study pages or something when they expire? Or only grey them out in the dedicated page not in news?

I would say edit them manually when they are not relevant anymore. I nominate you to update the posts from time to time ;-)

**Comment by adamperer** [comment]
> also paging @adamperer

Thanks for pushing this forward.  The plan sounds great.

**Comment by cabreraalex** [comment]
Alright, finally got laptop fixed and made the discussed changes. Definitely needs someone to test it out:

* New tab for news
* Studies in the news tab with "participate" link
* Dedicated studies page at /studies
* Some small changes like adding dates to news posts, renaming to news and adding link to main page

File: index.html [hunk 1]
```diff
@@ -74,11 +74,10 @@ <h3 class="f4 mt0">Design for Machine Learning</h3>
 
   <div class="flex flex-row-ns flex-column pv3">
     <div class="pr3 w-two-thirds-l w-50-m">
-      <h2 class="f3">Posts & Updates</h2>
+      <h2 class="f3">News</h2>
 
       <ul class="list pl0">
-        {% assign sorted_posts = site.posts %} {% for post in
-        sorted_posts %}
+        {% assign sorted_posts = site.posts %} {% for post in sorted_posts %}
```
Comment (domoritz) [inline index.html:80]: They are actually not sorted, lol.

File: index.html [hunk 2]
```diff
@@ -87,6 +86,7 @@ <h2 class="f3">Posts & Updates</h2>
         </li>
         {% endfor %}
       </ul>
+      <a class="cta" href="/news">Show all</a>
```
Comment (domoritz) [inline index.html:89]: Let's only add this if we restrict the number of news (which we should).

File: index.html (hunk 1 above)
Comment (domoritz) [reply index.html:80]:
```suggestion
        {% assign sorted_posts = site.posts %} {% for post in sorted_posts | limit:5 %}
```

File: news.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+subtitle: Check out our latest work, notable events, and ongoing studies!
+---
+
+<div>
+  {% for post in site.posts %}
+  <article class="pb2 measure-wide lh-copy">
+    <h2 class="mb2"><a href="{{ post.url }}">{{ post.title }}</a></h2>
+    <h4 class="mt0">{{ post.date | date: "%B %-d, %Y" }}</h4>
+    <p>{{ post.content }}</p>
+    {%if post.studyLink %}
```
Comment (domoritz) [inline news.html:13]:
```suggestion
    {% if post.studyLink %}
```

File: news.html [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    <a href="{{post.studyLink}}" class="black link hover-cmu-red mr3 dib">
+      <span class="cta">Participate</span>
+    </a>
+    {%endif%}
```
Comment (domoritz) [inline news.html:17]:
```suggestion
    {% endif %}
```

File: news.html [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+title: News
+subtitle: Check out our latest work, notable events, and ongoing studies!
+---
+
+<div>
+  {% for post in site.posts %}
+  <article class="pb2 measure-wide lh-copy">
+    <h2 class="mb2"><a href="{{ post.url }}">{{ post.title }}</a></h2>
+    <h4 class="mt0">{{ post.date | date: "%B %-d, %Y" }}</h4>
+    <p>{{ post.content }}</p>
```
Comment (domoritz) [inline news.html:12]:
```suggestion
    <p>{{ post.excerpt }}</p>
```

File: studies.html [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+subtitle: We work directly with data scientists to understand how they work and develop new tools and visualizations. We would love to work with you on any of our ongoing studies!
+---
+
+<div>
+  {% for post in site.posts %} {%if post.studyLink %}
+  <article class="pb2 measure-wide lh-copy">
+    <h2 class="mb2"><a href="{{ post.url }}">{{ post.title }}</a></h2>
+    <h4 class="mt0">{{ post.date | date: "%B %-d, %Y" }}</h4>
+    <p>{{ post.content }}</p>
+    <a href="{{post.studyLink}}" class="black link hover-cmu-red mr3 dib">
```
Comment (domoritz) [inline studies.html:13]:
```suggestion
    <a href="{{ post.studyLink }}" class="black link hover-cmu-red mr3 dib">
```

File: studies.html [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+      <span class="cta">Participate</span>
+    </a>
+  </article>
+  <hr />
```
Comment (domoritz) [inline studies.html:17]: Let's instead add some space.

File: news.html [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+    {%if post.studyLink %}
+    <a href="{{post.studyLink}}" class="black link hover-cmu-red mr3 dib">
+      <span class="cta">Participate</span>
+    </a>
+    {%endif%}
+  </article>
+  <hr />
```
Comment (domoritz) [inline news.html:19]: Let's instead add some space.

**Review by domoritz — CHANGES_REQUESTED** [review]
Looks great. I think we can make a few small improvements before we release.

File: news.html (hunk 8 above)
Comment (cabreraalex) [reply news.html:19]: added some but might be too much

Comment (domoritz) [reply news.html:19]: Did you look at other spacings on the website? I don't remember exactly the principle I used to determine spacing on the website but maybe you can find the pattern 😇.

File: _layouts/default.html [hunk 9]
```diff
@@ -48,6 +48,11 @@
                 href="{{ '/publications' | relative_url }}"
                 >Publications</a
               >
+              <a
```
[the remark this answers is not in the document]
Comment (cabreraalex) [reply _layouts/default.html:51]: Also centered the DIG logo on mobile, looks nicer imo

File: news.html (hunk 8 above)
Comment (cabreraalex) [reply news.html]: Wasn't really one, but made it smaller, #createthepattern

**Comment by domoritz** [comment]
This pull request introduces a regression: https://github.com/cmudig/cmudig.github.io/issues/63.

**Pull request merged**